tests/integration_tests/test_simple_features.py

```diff
@@ -162,21 +163,23 @@ def test_feature(input_test_feature, output_test_feature,
 @pytest.mark.parametrize(
     'input_test_feature, output_test_feature',
     [
-        ([category_feature()],
-         [binary_feature(), binary_feature()]),
-        ([category_feature()],
-         [category_feature(vocab_size=5), category_feature(vocab_size=7)]),
-        ([category_feature()],
-         [numerical_feature(), numerical_feature()]),
-        ([category_feature()],
-         [sequence_feature(vocab_size=5), sequence_feature(vocab_size=7)]),
-        ([set_feature(vocab_size=5)],
-         [set_feature(vocab_size=5), set_feature(vocab_size=7)]),
-        ([category_feature()],
-         [text_feature(vocab_size=5), text_feature(vocab_size=7)]),
-        ([category_feature()],
-         [vector_feature(), vector_feature()]),
-        ([vector_feature()],
+        # ([category_feature()],
```

Comment:
I'm guessing you were planning on removing the commented out tests before merging the PR?

Reply:
I've uncommented the tests (I think that's what you meant?) since we'll need them eventually.